libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTelemetry.java

```diff
@@ -110,6 +110,7 @@ public void onRouteProgressUpdate(RouteProgress routeProgress) {
 
   @Override
   public void onOffRouteEvent(Location offRouteLocation) {
+    updateDistanceCompleted();
     isOffRoute = true;
```

Comment:
Since we're setting `isOffRoute = true` here, do we want to add a guard at the top checking `isOffRoute == false` and returning early, so we don't accidentally call `updateDistanceCompleted()` and `queueRerouteEvent()` again if onOffRouteEvent is ever called more than once?

Reply:
Yeah good call -- will add and merge